pyi.py

```diff
@@ -2233,6 +2252,10 @@ def parse_options(options: argparse.Namespace) -> None:
     "class would be inferred as generic anyway"
 )
 Y061 = 'Y061 None inside "Literal[]" expression. Replace with "{suggestion}"'
+Y062 = (
+    'Y062 Argument "{arg}" to protocol method "{method}" should not be positional-or-keyword'
+    " (suggestion: make it positional-only)"
```

Comment:
```suggestion
    'Y062 Argument "{arg}" to protocol method "{method}" should probably not be positional-or-keyword. '
    " Make it positional-only, since usually you don't want to mandate a specific argument name"
```

Given that e.g. ruff will reimplement this error, can we soften the error message here